test/integration/services/data/test-get-claim-document-file-path.js

```diff
@@ -30,7 +30,7 @@ describe('services/data/get-claim-document-file-path', function () {
             return claimDocumentHelper.insert(REFERENCE, eligibilityId, claimId, claimDocumentHelper.DOCUMENT_TYPE)
           })
           .then(function (newClaimDocumentId) {
-            claimDocumentId = newClaimDocumentId
+            claimDocumentId = newClaimDocumentId[0].ClaimDocumentId
```

Comment:
Not relevant but fixed to ensure this change not causing failures. As is one below.